embark.el

```diff
@@ -666,7 +683,11 @@ when the indicator is no longer needed."
                           indicator)
                         target type
                         (if (cdr targets)
-                            (format "%S" (mapcar #'car (cdr targets)))
+                            ;; This is a weird feature of format/prin1-to-string.
+                            ;; A symbol list (function weird arbitrary symbols)
+                            ;; is printed as #'weird!
```

Comment:
I noticed!

Reply:
Great! I was really confused what is going on here. The usage of lists as function seems to be a weird design in elisp. Compiled functions should be opaque :laughing:

Reply:
Yes, it's very old-fashioned. I think many Lisps used to do that.